lua/metaworks/metabadges/server/modules/sitters.lua

```diff
@@ -0,0 +1,41 @@
+if not MetaBadges then return end
+
+local Tag = "MetaBadges"
+local id = "sitters"
+
+local levels = {
+	default = {
+		title = "Player Collector",
+		description = "Total number of players who have sat on your head."
+	}
+}
+
+MetaBadges.RegisterBadge(id, {
+	basetitle = "Player Collector",
+	levels = levels,
+	level_interpolation = MetaBadges.INTERPOLATION_FLOOR
+})
+
+hook.Add("PlayerEnteredVehicle", "%s_%s" % {Tag, id}, function(pl, veh)
```

Comment:
Keep in mind this is not valid syntax for lua and only works exclusively on meta.